client/src/pages/GalleryPage.js

```diff
@@ -10,31 +10,27 @@ const Display = styled.div`
 
 const GalleryPage = () => {
   const userName = "sven";
-  //const tag = "2";
-  const { data, loading, error, doFetch } = useAsync(() =>
+  const { data: userData, loading, error, doFetch } = useAsync(() =>
     getImageObj(userName)
   );
 
   useEffect(() => {
     doFetch();
   }, []);
 
-  const images = data[0].images;
-
   return (
     <>
       <div>
         <section>
           <h2>Das hier ist die Album-Seite 🤩</h2>
         </section>
         <Display>
-          <img src="" alt="testImage" />
           <p>Display für Cloudinary-Links von MongoDB</p>
           {loading && <p>Loading...</p>}
           {error && <p>{error.message}</p>}
-          {console.log(images)}
-          {data &&
-            images.map((image) => (
+          {console.log(userData)}
```

Comment:
console.logs are great for development, but should not end up in production.

Reply:
👍 You're right of course!